README.md

```diff
@@ -1,6 +1,6 @@
 # Database
 
-[![GitHub Actions CI](https://github.com/nmshd/cns-database/workflows/CI/badge.svg)](https://github.com/nmshd/cns-database/actions?query=workflow%3Apublish)
+[![GitHub Actions CI](https://github.com/nmshd/cns-database/workflows/Publish/badge.svg)](https://github.com/nmshd/cns-database/actions?query=workflow%3Apublish)
```

Comment:
Ich weiß, dass es auch so funktioniert, aber ich würde sagen du schreibst es hinten in der Query auch groß. Dann ist es einheitlich. Und als unbedarfter Leser versteht man leichter, was damit gemeint ist. :)